src/core/shaderlib/project/viewport-uniforms.js

```diff
@@ -124,14 +128,22 @@ export function getUniformsFromViewport({
     log.removed('positionOrigin', 'coordinateOrigin');
   }
 
+  return Object.assign(
+    {
```

Comment:
jspretty really doesn't do well with this pattern, which we unfortunately use a lot... sigh